e2e-polybft/e2e/storage_test.go

```diff
@@ -0,0 +1,150 @@
+package e2e
+
+import (
+	"math/big"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/jsonrpc"
+	"github.com/umbracle/ethgo/wallet"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+func TestE2E_Storage(t *testing.T) {
+	// premine an account in the genesis file
+	sender, err := wallet.GenerateKey()
+	require.NoError(t, err)
+
+	cluster := framework.NewTestCluster(t, 5,
+		framework.WithPremine(types.Address(sender.Address())),
+		framework.WithBurnContract(&polybft.BurnContractInfo{BlockNumber: 0, Address: types.ZeroAddress}),
+	)
+	defer cluster.Stop()
+
+	cluster.WaitForReady(t)
+
+	client := cluster.Servers[0].JSONRPC().Eth()
+
+	num := 20
+
+	receivers := []ethgo.Address{}
+
+	for i := 0; i < num; i++ {
+		key, err := wallet.GenerateKey()
+		require.NoError(t, err)
+
+		receivers = append(receivers, key.Address())
+	}
+
+	txs := []*framework.TestTxn{}
+
+	for i := 0; i < num; i++ {
+		func(i int, to ethgo.Address) {
+			txn := &ethgo.Transaction{
+				From:  sender.Address(),
+				To:    &to,
+				Gas:   30000, // enough to send a transfer
```

Comment:
21000 is always gas of a simple eoa transfer transaction.

```suggestion
				Gas:   21000, // enough to send a transfer
```